### PR TITLE
RAT-383 autoupdate ant documentation (Harmonize - Part 5 - pt 2) 

### PR DESCRIPTION
**Pull request by Claudenw**
Add Auto update to ANT documentation.

### CODE REVIEW

File: src/site/apt/index.apt.vm [hunk 1]
```diff
@@ -96,8 +96,11 @@ Apache Rat&#8482;
 
 * Quick Start: Running Rat
 
-  There are three user interfaces for Rat: from the command line, from Ant, and from Maven.  All three of the interfaces
```
Comment (ottlinger) [inline src/site/apt/index.apt.vm:99]: Shouldn't this information remain on the starting page of RAT?

**Comment by ottlinger** [comment]
Thanks for all your work - would you mind adding a changelog entry for Ant?
Apart from that minor things within the site need to be fixed. Thanks&&kudos!

File: apache-rat-tasks/src/site/apt/examples/basic.apt.vm [hunk 2]
```diff
@@ -59,28 +59,25 @@ java -jar apache-rat-${project.version}.jar src
...
+            editCopyright="Copyright 2008 Foo">
     <fileset dir="/path/to/project"/>
 </rat:report>
 ---
 
 * Create a styled HTML report
 
 ---
-<rat:report format="styled" reportFile="rat.html">
+<rat:report outputStyle="my/rat-xml-2-html.xslt" outputFile="rat.html">
```
[the remark this answers is not in the document]
Comment (Claudenw) [reply apache-rat-tasks/src/site/apt/examples/basic.apt.vm:79]: I am not certain what you are saying.  However here is how the outputStyle works.

- If not set use "plain-rat"
- May be set to one of the named XSLTs specified in the StyleSheets class:
    -  plain-rat - standard default.
    - missing-headers - enumerates files with missing headers
    - unapproved-licenses - files with unapproved licenses
    - xml - a pretty printed XML output.
 - My specify a file on the file system.

So I think we cover all the old "styled" optons, unless I am missing something.

File: src/site/apt/index.apt.vm (hunk 1 above)
Comment (Claudenw) [reply src/site/apt/index.apt.vm:99]: Isn't this the starting page of rat?

File: apache-rat-tasks/src/site/apt/examples/basic.apt.vm (hunk 2 above)
[the remark this answers is not in the document]
Comment (ottlinger) [reply apache-rat-tasks/src/site/apt/examples/basic.apt.vm:79]: I just looked at the diff - If I take a look at the old page, I see how I can define a styled report into rat.html.

The new version of the webpage contains a (somewhere?) defined/shown XSLT - to my mind we should either show that XSLT or provide an example without an XSLT .... that's what I tried to mention as a comment :) HTH?

File: src/site/apt/index.apt.vm (hunk 1 above)
Comment (Claudenw) [reply src/site/apt/index.apt.vm:99]: Ahhh  got it.  Good catch.  Sorry I was a bit slow.

File: src/site/site.xml [hunk 3]
```diff
@@ -61,6 +61,15 @@
       <item name="Option Name Cross Reference" href="apache-rat/name_xref.html" />
       <item name="Default Licenses, etc" href="apache-rat/default_licenses.html" />
       <item name="Configuration XSD" href="apache-rat/xsd.html" />
+      <item name="Default Configuration" href="https://github.com/apache/creadur-rat/blob/master/apache-rat-core/src/main/resources/org/apache/rat/default.xml" />
```
[the remark this answers is not in the document]
Comment (Claudenw) [reply src/site/site.xml]: fixed

File: apache-rat-tasks/src/site/apt/examples/basic.apt.vm [hunk 4]
```diff
@@ -59,28 +58,47 @@ java -jar apache-rat-${project.version}.jar src
...
+
+ The first is the standard report type.  This uses one of the standard reports shipped with Rat which are activated by setting the value of "outputStyled" to the name of the report.
+
+         *  plain-rat: The default style
+
+         * missing-headers: Produces a report of files that are missing headers
+
+         * unapproved-licenses: Produces a report of the files with unapproved licenses
+
+         * xml: The default style.
```
Comment (ottlinger) [inline apache-rat-tasks/src/site/apt/examples/basic.apt.vm:87]: As the default output is text I assume "xml" should have a different text or is the Ant version a combination of XML+RAT default report?

File: apache-rat-tasks/src/site/apt/examples/basic.apt.vm (hunk 2 above)
[the remark this answers is not in the document]
Comment (ottlinger) [reply apache-rat-tasks/src/site/apt/examples/basic.apt.vm:100]: Thanks for the enhanced docs!

File: apache-rat-tasks/src/site/apt/examples/basic.apt.vm (hunk 4 above)
Comment (Claudenw) [reply apache-rat-tasks/src/site/apt/examples/basic.apt.vm]: You are right.  It should say pretty printed XML.  I  see you updated the text.  I have updated the Sylesheet documentation in code.

**Comment by Claudenw** [comment]
@ottlinger  are we good to merge this PR?

File: src/site/apt/index.apt.vm (hunk 1 above)
Comment (ottlinger) [reply src/site/apt/index.apt.vm:99]: @Claudenw ping

File: src/site/apt/index.apt.vm [hunk 5]
```diff
@@ -96,8 +96,8 @@ Apache Rat&#8482;
...
-  support standard options as noted in the list below.
+  There are three user interfaces for Rat: from the command line, from Ant, and from Maven.
```
Comment (Claudenw) [inline src/site/apt/index.apt.vm:99]: @ottlinger Here is the changed code.

**Review by ottlinger — APPROVED** [review]
LGTM - any errors in the site or broken links will be fixed afterwards. Thanks again for the massive update to the ANT part of RAT.

**Pull request merged**